tests/MongoDB.Driver.Tests/Specifications/command-monitoring/CommandMonitoringTestRunner.cs

```diff
@@ -219,9 +234,39 @@ private void ExecuteOperation(IMongoDatabase database, IMongoCollection<BsonDocu
                 throw new SkipException(reason);
             }
 
+            var collection = GetCollection(database, collectionName, operation);
             test.Execute(__client.Cluster.Description, database, collection, arguments, async);
         }
 
+        private void ParseCollectionOptions(MongoCollectionSettings settings, BsonDocument collectionOptions)
```

Comment:
I think that these two `ParseCollectionOptions` methods can be combined into the single one.. But since these methods were copied from `CrudTestRunner`, I won't insist on it.

Reply:
I think these two `ParseCollectionMethod` methods are weird. This overload artificially takes a `settings` argument just to make it have a different signature than the other `ParseCollectionMethod`.

Better to use different method names that reflect how they are different.

This one can remain `ParseCollectionMethods` but should not take a `settings` argument. It should new up its own settings value, configure it, and return it.

The other one can be named `GetOperationCollectionOptions` or something like that.